cfgov/jinja2/v1/youth_employment_success/review/yes-info-unit.html

```diff
@@ -0,0 +1,25 @@
+{% from 'molecules/info-unit.html' import info_unit with context %}
+{{info_unit}}
+{% macro yes_info_unit(values) %}
+  {%- set body = '<p>' ~ values.body ~ '</p>' -%}
```

Comment:
Since you're not re-using them and they're pretty self-documenting, maybe ditch these variables and use the values directly where they are needed?